src/test/java/com/marizueva/laboratory/hw3/fluentpageobject/pages/HomePage.java

```diff
@@ -0,0 +1,171 @@
+    WebElement mainContentText;
+
+    @FindBy(xpath = "//iframe[@id='jdi-frame-site']")
+    WebElement iframeCenter;
+
+    @FindBy(xpath = "//img[@src='images/Logo_Epam_Color.svg']")
+    WebElement epamLogoInIframe;
+
+    @FindBy(xpath = "//div[@class='main-content']/h3[@class='text-center']")
+    WebElement subHeaderJdi;
+
+
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+        //        commonPageElements = new CommonPageElements(driver);
+    }
+
+    //    public CommonPageElements getCommonPageElements() {
+    ////        return commonPageElements;
+    //    }
+
+    public HomePage openSite() {
+        driver.navigate().to(siteUrl);
+        return this;
+    }
+
+    //    public String getBrowserTitle() {
+    //
+    //        return ;
+    //    }
+
+    public HomePage performLogin(String userName, String password) {
+        openLoginDropdownButton.click();
+        loginField.sendKeys(userName);
+        passwordField.sendKeys(password);
+        enterCredentials.click();
+        return this;
+    }
+
+    //    public HomePage checkLoggedUserName(String name) {
+    //
+    //        return this;
+    //    }
+    //
+    //    public HomePage checkHeaderItem(String headerItem){
+    //        return this;
+    //    }
+
+    public String getLoggedUserName() {
+        return loggedUserName.getText();
+    }
+
+    public String getHeaderItem(HeaderItems headerItem) {
+
+        switch (headerItem) {
+            case HOME:
+                return headerItemHome.getText();
+
+            case CONTACT_FORM:
+                return headerItemContact.getText();
+
+            case SERVICE:
+                return headerItemService.getText();
+
+            case METAL_AND_COLORS:
+                return headerItemMetalAndColor.getText();
+
+            default:
+                throw new NotFoundException("There is no such tab" + headerItem);
+
+        }
+    }
+
+
+    public boolean checkBenefitIconIsDisplayed(int index) {
+        switch (index) {
+            case 1:
+                return displayedBenefitImage1.isDisplayed();
+            case 2:
+                return displayedBenefitImage2.isDisplayed();
+            case 3:
+                return displayedBenefitImage3.isDisplayed();
+            case 4:
+                return displayedBenefitImage4.isDisplayed();
+            default:
+                throw new NotFoundException("there is no such icon");
+        }
+    }
+
+    public boolean checkBenefitTextIsDisplayed(int index) {
+        switch (index) {
+            case 1:
+                return benefitText1.isDisplayed();
+            case 2:
+                return benefitText2.isDisplayed();
+            case 3:
+                return benefitText3.isDisplayed();
+            case 4:
+                return benefitText4.isDisplayed();
+            default:
+                throw new NotFoundException("there is no such icon");
+        }
+    }
```

Comment:
`isBenefitTextDisplayed`? I do not see any checks in this method. Why do you call it `checkBenefitTextIsDisplayed` what do you check here?

Reply:
fixed